package.json

```diff
@@ -19,28 +19,28 @@
     "npm": ">=10.1.0"
   },
   "devDependencies": {
-    "@angular-builders/custom-webpack": "^15.0.0",
-    "@angular-devkit/build-angular": "^15.2.10",
-    "@angular/animations": "^15.2.10",
-    "@angular/cdk": "^15.2.9",
-    "@angular/cli": "^15.2.10",
-    "@angular/common": "^15.2.10",
-    "@angular/compiler": "^15.2.10",
-    "@angular/compiler-cli": "^15.2.10",
-    "@angular/core": "^15.2.10",
-    "@angular/forms": "^15.2.10",
-    "@angular/material": "^15.2.9",
-    "@angular/material-moment-adapter": "^15.2.9",
-    "@angular/platform-browser": "^15.2.10",
-    "@angular/platform-browser-dynamic": "^15.2.10",
-    "@angular/router": "^15.2.10",
+    "@angular-builders/custom-webpack": "^16.0.0",
+    "@angular-devkit/build-angular": "^16.2.14",
+    "@angular/animations": "^16.2.12",
+    "@angular/cdk": "^16.2.14",
+    "@angular/cli": "^16.2.14",
+    "@angular/common": "^16.2.12",
+    "@angular/compiler": "^16.2.12",
+    "@angular/compiler-cli": "^16.2.12",
+    "@angular/core": "^16.2.12",
+    "@angular/forms": "^16.2.12",
+    "@angular/material": "^16.2.14",
+    "@angular/material-moment-adapter": "^16.2.14",
+    "@angular/platform-browser": "^16.2.12",
+    "@angular/platform-browser-dynamic": "^16.2.12",
+    "@angular/router": "^16.2.12",
     "@angularclass/hmr": "^3.0.0",
     "@commitlint/cli": "^18.6.0",
     "@commitlint/config-conventional": "^18.6.0",
     "@compodoc/compodoc": "1.1.19",
     "@mdi/angular-material": "^4.0.96",
-    "@nationalbankbelgium/code-style": "^1.10.0",
-    "@nationalbankbelgium/eslint-config": "^15.0.0",
+    "@nationalbankbelgium/code-style": "^1.11.0",
+    "@nationalbankbelgium/eslint-config": "^16.0.0",
     "@ng-idle/core": "^13.0.1",
```

Comment:
Could you update the following dependencies to version compatible with Angular 16 ?

- "@ng-idle/core": "^14.0.0"
- "@ng-idle/keepalive": "^14.0.0"
- "@ngrx/effects": "^16.3.0"
- "@ngrx/store-devtools": "^16.3.0"
- "@ngrx/store": "^16.3.0"
- "@ngx-translate/core": "^15.0.0"